c10/util/Logging.h

```diff
@@ -305,6 +305,29 @@ namespace detail {
 C10_API bool LogAPIUsageFakeReturn(const std::string& context);
 } // namespace detail
 
+// Initializes the c10 logger.
+C10_API void initLogging();
+
+// The log levels for the c10 logger.
+enum class C10_API LogLevel {
+  Fatal   = 3,
+  Error   = 2,
+  Warning = 1,
+  Info    = 0
+};
+
+// Sets the threshold for the c10 logger to `level`.
+inline C10_API void setLogLevel(LogLevel level) noexcept {
```

Comment:
Any reason why this must be inline? I.e its probably called one or two times during the program lifetime

Reply:
No particular reason, simply because it was short and closely related to `getLogLevel` which is inline. I can move it to the implementation file in the next revision.